allure-framework-modules/allure-security-module/src/resources/settings.properties

```diff
@@ -0,0 +1,4 @@
+selenium.phantomjs=./lib/webdrivers/phantomjs/bin/phantomjs.exe
```

Comment:
Probably you do not use this "selenium. ...."  parameters. If yes, then remove it please.